index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link
```

Comment:
Could we move this to the repository? We try to avoid linking to assets like this since it may cause breakage if the hosting of those images changes (and we wouldn't know to update this project).